x-pack/plugins/observability_ai_assistant/public/routes/conversations/conversation_view.tsx

```diff
@@ -210,40 +214,39 @@ export function ConversationView() {
               })}
             </EuiCallOut>
           ) : null}
-          {chatService.loading || conversation.loading ? (
+          {chatService.loading ? (
```

Comment:
why is this gone?

Reply:
To make the loader that appears to push the ChatHeader down go away. Better fix welcome!

Reply:
I'll push a fix, no worries